jenkins/Jenkinsfile.nightly

```diff
@@ -188,9 +135,13 @@ pipeline {
 						script { echoBanner(STAGE_NAME) }
 						dir('lisk-service') {
 							nvm(readFile('.nvmrc').trim()) {
-								dir('./framework') { sh "npm run test:functional" }
-								dir('./services/blockchain-connector') { sh "npm run test:functional" }
+								// dir('./framework') { sh "npm run test:functional" }
+								dir('./services/blockchain-app-registry') { sh "npm run test:functional" }
+								// dir('./services/blockchain-connector') { sh "npm run test:functional" }
+								dir('./services/blockchain-indexer') { sh "npm run test:functional" }
+								dir('./services/fee-estimator') { sh "npm run test:functional" }
 								dir('./services/market') { sh "npm run test:functional" }
+								dir('./services/transaction-statistics') { sh "npm run test:functional" }
 								// dir('./tests') { sh "npm run test:functional" }
```

Comment:
Why do we have the functional tests disabled for the selected microservices? Can we instead disable the tests and leave it un-commented here?

Reply:
As discussed, let's keep it as is for now.